src/Vision/VNCircle.cs

```diff
@@ -0,0 +1,31 @@
+//
+// VNCircle.cs
+//
+// Authors:
+//	Alex Soto  <alexsoto@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation.
+//
+
+using System;
+using Foundation;
+using ObjCRuntime;
+
+namespace Vision {
+	public partial class VNCircle {
+
+		public static VNCircle CreateUsingRadius (VNPoint center, double radius)
+		{
+			var circle = new VNCircle (NSObjectFlag.Empty);
+			circle.InitializeHandle (circle.InitWithCenterRadius (center, radius));
+			return circle;
```

Comment:
Due to https://github.com/xamarin/xamarin-macios/issues/9478, this isn't a safe pattern. I think we'll have to do it the manual way:

```csharp
var handle = objc_msgSend (class_ptr, Selector.GetHandle ("alloc"));
handle = objc_msgSend_VNPoint_double (handle, Selector.GetHandle ("initWithCenterRadius:..."), center, radius);
return (VNCircle) Runtime.GetNSObject (handle, true):
```